README.md

```diff
@@ -1,2 +1,25 @@
-# toit-hx711
-Driver for the HX711 Analog-to-Digital converter
+# HX711
+
+A driver for the HX711 analog-to-digital converter often used for scales.
+
+This is a 24 bit D-to-A that uses a unique two-wire protocol with a clock
+and a data line.
+
+Keeping the clock high too long will cause a reset, which means it is somewhat
+timing sensitive.  This driver uses the UART circuitry to make sure the high
+clock pulses are not too long.  The time between the high clock pulses is not
+as critical.
+
+# Example
+
+```
+import hx711 show Hx711
+import gpio
+
+main:
+  hx711 := Hx711 --clock=17 --data=16
```

Comment:
I think we generally want to receive `gpio.Pin` as argument. (neither `--input` nor `--output`), but this is something we need to discuss a bit anyway.